reporters/kamon-prometheus/src/main/resources/reference.conf

```diff
@@ -82,6 +82,7 @@ kamon.prometheus {
     # Hostname and port used by the embedded web server to publish the scraping enpoint.
     hostname = "0.0.0.0"
     port = 9095
+    path = "/metrics"
```

Comment:
Maybe use `metrics-path` so it goes nicely with https://prometheus.io/docs/prometheus/latest/configuration/configuration/

Reply:
Nice catch, will do!

Reply:
`metrics-path` is more encouraged HOCON style I think. Sorry to bother you. :)

Reply:
Don't apologize, feedback is always welcome.

But I think I'm gonna stick with `_` on this one, Prometheus configuration uses metrics_path, so it makes sense if we're going for compatibility